manual/photon.xml.template

```diff
@@ -30,6 +30,22 @@
             <Label>DNS Domain</Label>
             <Description>DNS Domain</Description>
         </Property>
+        <Property ovf:key="guestinfo.enableproxy" ovf:type="string" ovf:userConfigurable="true">
+            <Label>Enable Proxy</Label>
+            <Description>Type yes or no</Description>
+        </Property>
+        <Property ovf:key="guestinfo.httpproxy" ovf:type="string" ovf:userConfigurable="true">
+            <Label>HTTP Proxy</Label>
+            <Description>Enter the address of your HTTP Proxy</Description>
+        </Property>
+        <Property ovf:key="guestinfo.httpsproxy" ovf:type="string" ovf:userConfigurable="true">
+            <Label>HTTPS Proxy</Label>
+            <Description>Enter the address of your HTTPS Proxy</Description>
+        </Property>
```

Comment:
Its not clear what the input should be here ... is it just proxy server (e.g. proxy.vmware.com) or is it the full proxy string (e.g. http://user:pass@proxy.vmware.com:port)? 

One option is passthrough which is telling user they must specify the proxy format OR breakup each component (HTTP vs HTTPs, Port, User and Pass) and then re-construct the final URL which is then passed.

The latter is better customer experience and just means you'll have a few more OVF properties

* Proxy Server (HTTP)
* Proxy Server (HTTPS)
* Proxy Server Port
* Proxy Username
* Proxy Password
* No Proxy